Project structure/Project setup.md

```diff
@@ -1,23 +1,12 @@
-## Before starting work
+## Repository setup
 
-1. Ask your Tech/Team leader to create a private Git repository on Github/Bitbucket.
-2. If you are going to use `release` branches, add protection regex with same rules as for the `main` branch.
-3. Ask the project manager to provide you with all project documentation and design.
+Ask a Lead Engineer to set up the repository on GitHub. If you're a Lead Engineer, follow the [Repository setup](https://github.com/infinum/LE-wiki/wiki/Repository-setup) guide on the LE wiki.
```

Comment:
Maybe we could suggest what protection rules to set up:
* Require a pull request before merging
* Require status checks to pass before merging with Require branches to be up to date before merging

Reply:
Added to the wiki 👍